README.rst

```diff
@@ -37,127 +37,117 @@ the Rescorla-Wagner learning rule to these corpora.
     This package is not intended to be used under python2.
 
 
-Installation
-============
-Install with::
+Getting Started
+===============
+These instructions will get you a copy of the project up and running on your
+local machine for development and testing purposes. See deployment for notes on
+how to deploy the project on a live system.
+
+
+Prerequisites
+-------------
+You need python 3.4 or newer and git installed on your machine. We recommend to
+install Minicoda (https://conda.io/miniconda.html) before installing ``pyndl``
+or to create a virtualenv within your personal folder.
+
+
+Installing
+----------
+If you only want to use the package run you can install ``pyndl`` from pypi with:
 
-    python setup.py install [--user]
+.. code:: bash
 
-or development install with::
+    pip install --user pyndl
 
+If you want to inspect and change the source code as well as running tests and
+having the documentation at hand clone the repository and install the package
+in 'development' mode by running
+
+.. code:: bash
+
+    git clone https://github.com/quantling/pyndl.git
+    cd pyndl
     python setup.py develop
 
-in order to build a source package run::
 
-    python setup.py sdist
+Documentation and Examples
+--------------------------
+Documentation and examples can be found under
+http://pyndl.readthedocs.io/en/latest/ or in the ``doc/`` folder after cloning
+the repository.
 
 
-Usage
-=====
-This package is intended to be used as a python package in (small) python
-script or via the ipython REPL. For example code and explanations look into
-``doc/source/examples.rst``.
+Running the tests
+=================
+If you have installed ``pyndl`` by cloning the repository change directory to
+the repository and run (for fast tests):
 
+.. code:: bash
 
-Development
-===========
+    cd pyndl
+    py.test
+    py.test doc/souce/example.rst
```

Comment:
Usually in the README are informations for developers and endusers. 
CONTRIBUTING.rst is usually more about organisational stuff and rights  like PR management, acknowledgement ,...